java/checkstyle/oilmod-checkstyle.xml

```diff
@@ -3,6 +3,13 @@
         "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
         "https://checkstyle.org/dtds/configuration_1_3.dtd">
 
+<!-- 
+    This configuration has been adjusted from the original Google Checkstyle configuration,
+    which can be found at https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml.
+
+    For more information, please refer Oilmod code style related article in Wiki.
```

Comment:
This is a public repository, the wiki article referred to is surely only internal?

Reply:
Removed reference in PR: https://github.com/equinor/oilmod1-code-style/pull/15